src/components/DepositWidget/useRowActions.tsx

```diff
@@ -45,8 +47,16 @@ export const useRowActions = (params: Params): Result => {
   }
 
   async function enableToken(tokenAddress: string): Promise<void> {
-    const { symbol } = getToken('address', tokenAddress, balances)
     try {
+      assert(userAddress, 'User address missing. Aborting.')
+      assert(contractAddress, 'Contract address missing. Aborting.')
+
+      const token = getToken('address', tokenAddress, balances)
+      assert(token, 'No token, aborting.')
+
+      const { symbol: tokenDisplayName } = safeFilledToken(token)
+      assert(tokenDisplayName, 'No token name found - symbol, name, or address. Aborting.')
```

Comment:
Address is required and should never fail. I mean, should fail sooner in case token has no address...

Reply:
yes true